addons/web/static/src/views/form/form_controller.scss

```
@@ -42,8 +42,10 @@
             }
 
             .o_field_widget {
-                width: 100%;
                 margin-bottom: $o-form-spacing-unit * 2;
```

Add an empty row before nested selectors

```suggestion
                margin-bottom: $o-form-spacing-unit * 2;
                
```

---

Thanks for the reviews!